### PR TITLE
Fix robotiq FTS ros2 driver

### PR DESCRIPTION
**Pull request by pac48**
This PR closes https://github.com/PickNikRobotics/moveit_studio/issues/7047. I tested on the hardware and the robot is running like it used to. I moved the sensor reading off of the real-time thread.

### CODE REVIEW

**Comment by pac48** [comment]
@dyackzan Should I break this into two PRs? One for running pre-commit and one for this change?

**Comment by pac48** [comment]
@dyackzan This PR formats the code https://github.com/PickNikRobotics/rq_fts_ros2_driver/pull/2

File: .clang-format
Comment (dyackzan) [inline .clang-format:1]: Hmm looks like this one still has the formatting changes in it

File: .pre-commit-config.yaml
Comment (dyackzan) [inline .pre-commit-config.yaml:1]: Also this file, shouldn't this just get added in the "Add pre-commit" PR?

**Review by dyackzan — COMMENTED** [review]
I like your idea to separate the formatting changes from the functional changes but I think you accidentally got them both tangled up here. Could you weed out just the functional changes in this PR?

**Comment by pac48** [comment]
@dyackzan It should be all set now.

**Review by dyackzan — APPROVED** [review]

**Pull request merged**